.travis.yml

```diff
@@ -0,0 +1,25 @@
+dist: xenial
+sudo: required
+
+language: generic
+
+addons:
+  apt:
+    packages:
+     - build-essential
+     - make
+     - perl
+     - git
+     - unzip
+     - m4
+     - gcc
+     - curl
+     - libpcre3
+     - libpcre3-dev
+     - zlib1g-dev
```

Comment:
We should try reduce this list as much as possible. Maybe we can try:
```
- build-essential
- perl
- curl
```

libpcre can also be added, with our jobs ensuring that it picks up the system's lib, if not specified as the `--pcre` argument to `kong-ngx-build`.

Reply:
In a docker starting from xenial I didn't manage to get a successful build until I installed all these packages. Will try again

Reply:
We build ngx_lua from source with many less packages, so worth a try imho.

Reply:
True, the base docker image != generic travis, my bad